src/DeviceDriver/DeviceDriver.cpp

```diff
@@ -36,7 +36,7 @@ vr::EVRInitError DeviceDriver::Activate(uint32_t unObjectId) {
           IsRightHand() ? "/input/skeleton/right" : "/input/skeleton/left",
           IsRightHand() ? "/skeleton/hand/right" : "/skeleton/hand/left",
           "/pose/raw",
-          vr::VRSkeletalTracking_Partial,
+          vr::EVRSkeletalTrackingLevel::VRSkeletalTracking_Full,
```

Comment:
Is there any reason why we might want to leave this on partial while we aren't doing full joint stuff?
If so then it definitely could be a setting. Otherwise, if there's no reason to then there's no need.

Reply:
We are pretty much doing full join things now, you can move the individual joints, and splay them

Reply:
Yes, but I mean, for people whose gloves don't support invividual joint and splay (which is 99.9% of people), this is not the case. For them, is there any compatibility advantage to keeping it set at partial? Or is it just a metadata thing for stats and stuff

Reply:
I'm pretty sure it's used as a hint, and doesn't really mean anything... I don't think there's any harm in changing it, especially as partial wouldn't encompass all the splay and multi joint gloves

Reply:
Yeah, cool sounds good then. My original thought was to have a setting for it so that people with multi joint could set it to full, but if it doesn't have any effect on compatibility or performance than might as well leave it this way.